lib/alertmanager_api/silence_details.py

```diff
@@ -0,0 +1,265 @@
+import json
+from datetime import datetime
+
+
+class SilenceDetails:  # pylint: disable=too-few-public-methods
+    def __init__(
+        self,
+        instance_name=None,
```

Comment:
what's this for - hypervisors? 
Makes this SilenceDetails struct only useful for hypervisor related silences - lets remove this to make the struct more generic - so we can use it for silencing other types of alerts in the future